packages/e2e/suites/full/landing.spec.ts

```diff
@@ -27,14 +29,14 @@ test("TC103 - Login as zk foundation", async ({ switcher, page }) => {
   await expect(page.getByTestId("user-role")).toHaveText("ZkSync Foundation");
 });
 
-test("TC104 - View all buttons in private app", async ({ page }) => {
+test("TC104 - View all buttons in private app", async ({ page, testApp }) => {
   await expect(page.getByText("Standard Upgrades")).toBeVisible();
   await expect(page.getByText("Emergency Upgrades")).toBeVisible();
   await expect(page.getByText("Freeze Requests")).toBeVisible();
   await expect(page.getByText("L2 Proposals Veto")).toBeVisible();
 });
 
-test("TC105 - View only standard upgrades in private app", async ({ page }) => {
+test.skip("TC105 - View only standard upgrades in private app", async ({ page, testApp }) => {
```

Comment:
```suggestion
test("TC105 - View only standard upgrades in private app", async ({ page, testApp }) => {
```

Reply:
Done, but surrounding the test in all kind of security measuers to avoid fake negatives